packages/flutter/test/cupertino/refresh_test.dart

```diff
@@ -325,15 +325,15 @@ void main() {
           mockHelper.builder(
             any,
             RefreshIndicatorMode.armed,
-            150.0,
+            132.08099300425673,
             100.0, // Default value.
             60.0, // Default value.
           ),
           mockHelper.refreshTask(),
           mockHelper.builder(
             any,
             RefreshIndicatorMode.armed,
-            argThat(moreOrLessEquals(127.10396988577114)),
+            argThat(moreOrLessEquals(111.92012371530531)),
```

Comment:
this is a great example. I have no way to review this to know if it's correct or not.